libraries/SITL/SIM_Aircraft.cpp

```diff
@@ -1058,6 +1067,9 @@ void Aircraft::update_external_payload(const struct sitl_input &input)
 
     {
         const float range = rangefinder_range();
+        if (!isinf(range) && range > 100000) {
+            AP_HAL::panic("Bad rangefinder calculation");
```

Comment:
What happens if a PANIC occurs during flight?
I think it's unacceptable if it becomes uncontrollable.

Reply:
that is SITL, panic will just block the simulation to allow to catch the bug